sheepdog_exporter/sheepdog_exporter.py

```diff
@@ -218,7 +218,12 @@ def get_submissions_by_type(self, program, project, my_type):
         '''
         url = '{}/{}/{}/export/?node_label={}&format=json'.format(self.sheep_url, program, project, my_type)
         response = requests.get(url, headers=self.headers())
-        return response.json()['data']
+        try:
+            submissions = response.json()['data']
+        except Exception as e:
```

Comment:
Suggest except on KeyError instead of broad `Exception`.

Also probably this should go the stderr?